### DISCUSSION TITLE
RPC endpoint fails to fetch on wallet connect

### DISCUSSION THREAD

**Discussion by Bagdu** [WalletConnect/walletconnect-monorepo#3966] [Developer Support] [answered]
This is `WalletConnect` `projectId` we've used on our product: `4e8a3f79afbffa3cc20950b38dcff42a` . When we try to connect metamask account with gnosis chain on our website, and makes some interaction on RPC the `WalletConnect` fails to fetch. see screens below. Our website is on this https://app.protobox.xyz/
<img width="315" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/39549813/1922d784-fcf4-4b3d-8daf-b4ad72a87961">
<img width="825" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/39549813/f1ea6f87-a312-43e4-9401-1ab1968d9b35">
2-3 days ago this endpoints has been worked without problem.

**Comment by billionsQuest** [comment]
We are encountering a similar issue with our transaction process as before. This problem is causing our users to halt their transactions. i guess the rpc is down of the walletconnect
<img width="568" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/147618200/7f3006ec-d055-4e36-9feb-5f3ffac27e23">

**Reply by Bagdu** [reply]
on `Goerli`, `gnosis`, `Ethereum` I have same problems, i think it `WalletConnect` server's issue

**Comment by dan-supetree** [comment]
here we also encountering same issue...

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/72896580/d2bd9a76-c821-4358-96f7-8252f41c8d20)

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/72896580/7312d8eb-2499-48a4-b222-a8a1a304d95f)

**Reply by chusla** [reply]
I have multiple clients reporting the same thing.  Working couple days ago and now not. Any color from wallet connect team?

**Comment by oeroche** [comment]
Same issue on our side, can't the server returns empty responses for getTransactionByHash rpc calls.

**Comment by geekbrother** [comment, marked as answer]
There was a [partial outage in API](https://status.walletconnect.com/incidents/gxx0g5nmjqwm) which is resolved for now, apologize for the inconvenience 🙏
